app/controllers/users.js

```diff
@@ -44,6 +44,99 @@ exports.findUser = (req, res) => {
   });
 };
 
+exports.getFriends = (req, res) => {
+  const userId = req.body.userId;
+  try {
```

Comment:
Why the use of a try catch block around functions with callbacks. We should return responses with appropriate status codes and message when there is a success or error.